.github/workflows/unit_test.yml

```diff
@@ -13,5 +13,7 @@ jobs:
         run: |
           gem install bundler
           bundle install --jobs 4 --retry 3
+      - name: Run tests
+        run: rspec ./spec/unit/xml_params_spec.rb
```

Comment:
Never run a single spec, you defenetly to forgot add new spec in config after you add new spec for some reason

Use
```
rspec spec/unit
```